pkgs/tools/filesystems/zerofree/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, e2fsprogs }:
+
+stdenv.mkDerivation rec {
+  name = "zerofree-1.0.3";
+
+  src = fetchurl {
+    url = "http://intgat.tigress.co.uk/rmy/uml/zerofree-1.0.3.tgz";
+    sha256 = "3acfda860be0f0ddcb5c982ff3b4475b1ee8cc35a90ae2a910e93261dbe0ccf6";
+  };
+
+  buildInputs = [ e2fsprogs ];
+
+  installPhase = ''
+mkdir -p $out/bin
```

Comment:
Ah. Thanks. Looks like I didn't guess the style correctly. Looking at the contributor guide I see those multi-line strings all indented based on the surrounding expression.

My background is Python and I'm used to make multi-line strings start indenting by 0 to make them readable if they appear standalone (e.g. looking at the file that is generated).

Anyway: thanks for the catch. Will update.